src/SprykerEco/Zed/PunchoutCatalogs/Communication/Form/PunchoutCatalogCXmlConnectionFormatForm.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+/**
+ * Copyright © 2016-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerEco\Zed\PunchoutCatalogs\Communication\Form;
+
+use Generated\Shared\Transfer\PunchoutCatalogConnectionTransfer;
+use Spryker\Zed\Kernel\Communication\Form\AbstractType;
+use Symfony\Component\Form\Extension\Core\Type\TextType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\Validator\Constraints\NotBlank;
+
+/**
+ * @method \SprykerEco\Zed\PunchoutCatalogs\PunchoutCatalogsConfig getConfig()
+ * @method \SprykerEco\Zed\PunchoutCatalogs\Persistence\PunchoutCatalogsRepositoryInterface getRepository()
+ * @method \SprykerEco\Zed\PunchoutCatalogs\Business\PunchoutCatalogsFacadeInterface getFacade()
+ * @method \SprykerEco\Zed\PunchoutCatalogs\Communication\PunchoutCatalogsCommunicationFactory getFactory()
+ */
+class PunchoutCatalogCXmlConnectionFormatForm extends AbstractType
+{
+    protected const FIELD_LABEL_SENDER_ID = 'Sender ID';
+    protected const FIELD_LABEL_SHARED_SECRET = 'Shared Secret';
+
+    /**
+     * @param \Symfony\Component\Form\FormBuilderInterface $builder
+     * @param array $options
+     *
+     * @return void
+     */
+    public function buildForm(FormBuilderInterface $builder, array $options): void
+    {
+        $this->addUsernameField($builder)
+            ->addPasswordField($builder);
+    }
+
+    /**
+     * @param \Symfony\Component\Form\FormBuilderInterface $builder
+     *
+     * @return $this
+     */
+    protected function addUsernameField(FormBuilderInterface $builder)
+    {
+        $builder->add(PunchoutCatalogConnectionTransfer::USERNAME, TextType::class, [
+            'label' => static::FIELD_LABEL_SENDER_ID,
+            'constraints' => [
+                new NotBlank(),
+            ],
+        ]);
+
+        return $this;
+    }
+
+    /**
+     * @param \Symfony\Component\Form\FormBuilderInterface $builder
+     *
+     * @return $this
+     */
+    protected function addPasswordField(FormBuilderInterface $builder)
```

Comment:
- `addPasswordField()`
- `FIELD_LABEL_SHARED_SECRET`

Please keep naming consistency, and stick to one way, or to another.